src/foam/mlang/mlang.js

```diff
@@ -1508,10 +1508,11 @@ foam.CLASS({
 
         Object[] a = null;
         Object arg2 = getArg2().f(null);
```

Comment:
please add null check on this line

Reply:
for the null case it would jump to line 1525 which doesn't access arg2 so I think we are fine.

Reply:
i mean if getArg2() is null, would that ever be the case?

Reply:
yeah, getArg2() can be null but it would not throw. The toString() would just return empty arg2 eg. `In(arg1, )`.

Reply:
nvm just tried, i thought getArg2().f() would throw on null arg2 but it just returned constant. so lgtm, thanks!